Tests/SentryTests/UIRedactBuilderTests.swift

```diff
@@ -394,6 +405,32 @@ class UIRedactBuilderTests: XCTestCase {
             XCTAssertTrue(sut.containsIgnoreClass(element), "\(element) not found")
         }
     }
+    
+    func testLayerIsNotFullyTransparentRedacted() {
+        let sut = getSut()
+        let view = CustomVisibilityView(frame: CGRect(x: 20, y: 20, width: 40, height: 40))
+        view.alpha = 0
+        view.sentryReplayMask()
+        
+        view.backgroundColor = .purple
+        rootView.addSubview(view)
+        
+        let result = sut.redactRegionsFor(view: rootView)
+        XCTAssertEqual(result.count, 1)
+    }
+    
+    func testViewLayerOnTopIsNotFullyTransparentRedacted() {
+        let sut = getSut()
+        let view = CustomVisibilityView(frame: CGRect(x: 20, y: 20, width: 40, height: 40))
+        let label = UILabel(frame: CGRect(x: 20, y: 20, width: 40, height: 40))
+        view.backgroundColor = .purple
+        rootView.addSubview(label)
+        rootView.addSubview(view)
+        
+        let result = sut.redactRegionsFor(view: rootView)
+        XCTAssertEqual(result.first?.type, .redact)
+        XCTAssertEqual(result.count, 1)
```

Comment:
`l`: I don't understand how we can be sure that we actually redacted the label with these assertions. We only check if there's a redacted region. How can we be sure the whole label was redacted?